### PR TITLE
Пикин Илья Сергеевич | next

### PR DESCRIPTION
**Pull request by IlyaPikin**

### CODE REVIEW

File: include/tbitfield.h [hunk 1]
```diff
@@ -13,24 +13,27 @@ typedef size_t elType;
 class TBitField
 {
 private:
-    size_t bitLen;   // длина битового поля - макс. к-во битов
-    elType* pMem;     // память для представления битового поля
-    size_t memLen;   // к-во эл-тов elType для представления бит.поля //длина массива pMem
+    size_t bitLen = 0;   // длина битового поля - макс. к-во битов
+    elType *pMem = 0;    // память для представления битового поля
```
Comment (AleksandrPanov) [inline include/tbitfield.h]: use `nullptr`

File: src/tbitfield.cpp [hunk 2]
```diff
@@ -9,30 +9,49 @@
 #include <iostream>
 #include "tbitfield.h"
 
-TBitField::TBitField(size_t len)
-{
 
+
+size_t bitLen = 0;   // длина битового поля - макс. к-во битов
+elType* pMem = 0;      // память для представления битового поля
```
Comment (AleksandrPanov) [inline src/tbitfield.cpp]: use `nullptr`
dublicated in `include/tbitfield.h`

File: include/tbitfield.h (hunk 1 above)
Comment (IlyaPikin) [reply include/tbitfield.h]: Ok

**Comment by AleksandrPanov** [comment]
:+1:

File: src/tbitfield.cpp (hunk 2 above)
Comment (IlyaPikin) [reply src/tbitfield.cpp]: fixed nullptr

**Pull request closed without merging**